examples/ft-refund/integration-tests/src/lib.rs

```diff
@@ -0,0 +1,331 @@
+#[cfg(test)]
+mod tests {
+    use aurora_sdk_integration_tests::{
+        aurora_engine::{self, AuroraEngine, ContractInput},
+        aurora_engine_sdk::types::{self, near_account_to_evm_address},
+        aurora_engine_types::{
+            types::{Address, Wei},
+            U256,
+        },
+        ethabi, tokio,
+        utils::{self, ethabi::DeployedContract, forge},
+        wnear,
+        workspaces::{self, Account, AccountId},
+    };
+    use std::path::Path;
+
+    #[tokio::test]
+    async fn test_contract() {
+        let worker = workspaces::sandbox().await.unwrap();
+        let engine = aurora_engine::deploy_latest(&worker).await.unwrap();
+        let wnear = wnear::Wnear::deploy(&worker, &engine).await.unwrap();
+
+        // Deploy Near contract
+        tokio::fs::create_dir_all("../target/near/ft_refund")
+            .await
+            .unwrap();
+        tokio::fs::create_dir_all("../target/near/test_token")
+            .await
+            .unwrap();
+        let contract_bytes = utils::cargo::build_contract("../near-contract")
+            .await
+            .unwrap();
+        let near_contract = worker.dev_deploy(&contract_bytes).await.unwrap();
+        let contract_bytes = utils::cargo::build_contract("../test-token").await.unwrap();
+        let test_token_contract = worker.dev_deploy(&contract_bytes).await.unwrap();
+
+        let account = worker.dev_create_account().await.unwrap();
+        let address = near_account_to_evm_address(account.id().as_bytes());
+
+        let test_erc20 = engine
+            .bridge_nep141(test_token_contract.id())
+            .await
+            .unwrap();
+
+        // Deploy solidity contract
+        let solidity_contract =
+            deploy_solidity_contract(&engine, near_contract.id(), wnear.aurora_token.address).await;
+
+        let xcc_account = format!(
+            "{}.{}",
+            solidity_contract.inner.address.encode(),
+            engine.inner.id()
+        );
```

Comment:
I think it is good for users to understand the xcc internals to some extent. The account created by the engine becomes their identity on Near, so it could be valuable to know the account id.

That said, it might still make sense to have a helper function for this just for convenience. But I think that is out of scope for this PR.

Reply:
Agreed, adding this helper function would better be added in a separate PR.